modules/ROOT/pages/6.0.3-release-notes.adoc

```diff
@@ -0,0 +1,94 @@
+= TinyMCE 6.0.3
+:navtitle: TinyMCE 6.0.3
+:description: Release notes for TinyMCE 6.0.3
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.0.3 was released for {enterpriseversion} and {cloudname} on Wednesday, June 1^st^, 2022. It includes {productname} 6.0.3. These release notes provide an overview of the changes for {productname} 6.0.3:
+
+* xref:general-bug-fixes[General bug fixes]
+* xref:known-issues[Known issues]
+
+[[general-bug-fixes]]
+== General bug fixes
+
+{productname} 6.0.3 provides fixes for the following bugs:
+
+=== When multiple cells were selected, removing values in the cell properties dialog did not work
+
+Previously, when dealing with {productname} tables with two or more cells, the cell properties dialog box did not honour deleting values when multiple cells were selected.
+
+If multiple table cells were selected in a {productname} table with two or more columns and the cell properties dialog was invoked, deleting the `height` value in the dialog box had no effect. However, if each cell was selected individually, invoking the cell properties dialog and deleting the `height` value worked as expected.
+
+With this update, selecting multiple cells, invoking the cell properties dialog, and removing `height` values across the selected cells works as expected, with all cell heights deleted at once.
+
+=== When multiple rows were selected, removing values in the row properties dialog did not work
+
+Previously, when dealing with {productname} tables with two or more rows, the row properties dialog box did not honour deleting values when multiple rows were selected.
+
+If multiple table rows were selected in a {productname} table with two or more columns and the cell properties dialog was invoked, deleting the height value in the dialog box had no effect. However, if each cell was selected individually, invoking the cell properties dialog and deleting the height value worked as expected.
```

Comment:
```suggestion
If multiple table rows were selected in a {productname} table with two or more columns and the cell properties dialog was invoked, deleting, for example, the `height` value in the dialog box had no effect. However, if each cell was selected individually, invoking the cell properties dialog and deleting the `height` value worked as expected.
```

Reply:
Added the same sub-clause. Also caught my own typos: had `height` marked up in the equivalent paragraph above, but not this one.